static/js/settings_users.js

```diff
@@ -199,6 +199,13 @@ function populate_users(realm_people_data) {
             return $("<span></span>").text(i18n.t("Unknown"));
         }
         if (item.last_active === LAST_ACTIVE_NEVER) {
+            // only two possibilities remain either user has never logged in
+            // or the user was last active more than 2 weeks ago. Check this
+            // by getting data from same place as the sidebar user list
+            const title_data = buddy_data.get_title_data(item.user_id, false);
+            if (title_data.second_line === "Last active: More than 2 weeks ago") {
```

Comment:
This should probably be `title_data.last_seen`.

Reply:
@vinitS101 thanks for reviewing! I have edited the commit message. It would be great if you could suggest any further changes to it.

Regarding code, `title_data.second_line` works too. I edited the User Activity of a few users and tested it. If you want, I can change it to 'title_data.last_seen' .

Reply:
Yeah, you should probably use that since this might be a little confusing for someone else reading the code on a later date.

The `title_data` object has no `second_line` attribute, and even though I wrote that code, I was wondering where this came from (and if it even worked).

Reply:
I don't think this check will work with internationalization, because the `second_line` is already translated into the target language.  We need to design a test that isn't based on parsing strings.

(Probably involves changing `get_title_data`).